File: app/src/main/java/com/manimarank/spell4wiki/models/WikiLanguage.java
package com.manimarank.spell4wiki.models;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class WikiLanguage {

    @SerializedName("code")
    @Expose
    private String code;

    @SerializedName("lang")
    @Expose
    private String name;

    @SerializedName("dir")
    @Expose
    private String direction;
    @SerializedName("local_lang")
    @Expose
    private String localName;

    @SerializedName("title_words_without_audio")
    @Expose
    private String titleOfWordsWithoutAudio;

    public String getCode() {
        return code;
    }

    public void setCode(String code) {
        this.code = code;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getDirection() {
        return direction;
    }

    public void setDirection(String direction) {
        this.direction = direction;
    }

    public String getLocalName() {
        return localName;
    }

    public void setLocalName(String localName) {
        this.localName = localName;
    }

    public String getTitleOfWordsWithoutAudio() {
        return titleOfWordsWithoutAudio;
    }

    public void setTitleOfWordsWithoutAudio(String titleOfWordsWithoutAudio) {
        this.titleOfWordsWithoutAudio = titleOfWordsWithoutAudio;
    }
}

File: app/src/main/java/com/manimarank/spell4wiki/data/model/WikiLanguage.kt
package com.manimarank.spell4wiki.data.model

import com.google.gson.annotations.SerializedName

/**
 * Model Class for Wiki Language
 * @property code String?
 * @property name String?
 * @property direction String?
 * @property localName String?
 * @property titleOfWordsWithoutAudio String?
 * @property category List<String>?
 * @constructor
 */
data class WikiLanguage(
    @SerializedName("code")
    var code: String? = null,
    @SerializedName("lang")
    var name: String? = null,
    @SerializedName("dir")
    var direction: String? = null,
    @SerializedName("local_lang")
    var localName: String? = null,
    @SerializedName("title_of_words_list")
    var titleOfWordsWithoutAudio: String? = null,
    @SerializedName("category")
    var category: List<String>? = null
)